static/app/components/events/eventViewHierarchy.tsx

```diff
@@ -16,53 +14,25 @@ import {ViewHierarchy, ViewHierarchyData} from './viewHierarchy';
 const DEFAULT_RESPONSE: ViewHierarchyData = {rendering_system: '', windows: []};
 const FIVE_SECONDS_IN_MS = 5 * 1000;
 
-function fillWithUniqueIds(hierarchy) {
-  return {
-    ...hierarchy,
-    id: uniqueId(),
-    children: hierarchy.children?.map(fillWithUniqueIds) ?? [],
-  };
-}
-
 type Props = {
   projectSlug: string;
   viewHierarchies: EventAttachment[];
 };
 
 function EventViewHierarchy({projectSlug, viewHierarchies}: Props) {
   const [selectedViewHierarchy] = useState(0);
-  const api = useApi();
   const organization = useOrganization();
 
   const hierarchyMeta = viewHierarchies[selectedViewHierarchy];
-  const {isLoading, data} = useQuery<ViewHierarchyData>(
-    [`viewHierarchies.${hierarchyMeta.id}`],
-    async () => {
-      const response = await api.requestPromise(
-        getAttachmentUrl({
-          attachment: hierarchyMeta,
-          eventId: hierarchyMeta.event_id,
-          orgId: organization.slug,
-          projectId: projectSlug,
-        }),
-        {
-          method: 'GET',
-        }
-      );
-
-      if (!response) {
-        return DEFAULT_RESPONSE;
-      }
-
-      const JSONdata = JSON.parse(response);
-
-      return {
-        rendering_system: JSONdata.rendering_system,
-        // Recursively add unique IDs to the nodes for rendering the tree,
-        // and to correlate elements when hovering between tree and wireframe
-        windows: JSONdata.windows.map(fillWithUniqueIds),
-      };
-    },
+  const {isLoading, data} = useQuery<string>(
```

Comment:
since `data` is now a string, I don't think `isEqual(DEFAULT_RESPONSE, data)` will ever be true (from line 40). Do we need to update `DEFAULT_RESPONSE` to a string? or maybe we can do the `JSON.parse` before line 40.

Reply:
also should we wrap the `JSON.parse` in a try catch just to be safe?

Reply:
I think I can actually remove the `DEFAULT_RESPONSE` check because it was relevant earlier when I messed up conditionally rendering this, and it was possible for the hook to run and not get anything back so I set a default

I'll add a try/catch around `JSON.parse` though